src/main/java/com/paypal/security/ApplicationConfiguration.java

```diff
@@ -30,11 +30,12 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public class SecurityConfiguration {
+public class ApplicationConfiguration {
 
-  public static final Logger LOG = LoggerFactory.getLogger(SecurityConfiguration.class.getName());
+  public static final Logger LOG =
+      LoggerFactory.getLogger(ApplicationConfiguration.class.getName());
 
-  private static final String SEC_PROPERTIES = "security.properties";
+  private static final String SEC_PROPERTIES = "application.properties";
```

Comment:
`SEC_PROPERTIES` -> `APP_PROPERTIES` is probably needed too.

Reply:
Agreed!